plugin/src/test/sc/plugin2021/GameTest.kt

```diff
@@ -71,6 +71,7 @@ class GameTest: WordSpec({
                 
                 shouldNotThrowAny {
                     while (!game.checkGameOver()) {
+                        Game.logger.debug("is Game Over? ${game.isGameOver} - ${state.round} > ${Constants.ROUND_LIMIT}")
```

Comment:
A class should ALWAYS use its own logger.

Reply:
fair point
622a3253

Reply:
actually, I removed the logging statement. That was only used for debugging the `isGameOver` function